data.json

```diff
@@ -179,6 +179,15 @@
     "dateAdded": "2018-10-23T10:17:39.000Z",
     "tags": ["clothing", "hacktoberfest", "expired"]
   },
+  {
+    "name": "MayaData.io",
```

Comment:
MayaData not MayaData.io :)

Reply:
I think I fixed that, could you check again?

Reply:
👍